src/operator/nn/mkldnn/mkldnn_rnn.cc

```diff
@@ -47,6 +47,14 @@ inline int GetRnnGatesNum(int mode) {
   }
 }
 
+// Bug in oneDNN >= 1.6  in memory descriptor comparision operators.
+// for specific dims and strides in descriptors == operator can return `true`
+// but get_size() function will return different size
+// TODO(bgawrych): Remove with oneDNN 1.7 upgrade
+bool CheckMemDescEquality(const mkldnn::memory::desc &left, const mkldnn::memory::desc &right) {
```

Comment:
better to make this function as `static inline`.

Reply:
done :)